pkg/internal/api/clickpipe_models.go

```diff
@@ -144,12 +144,14 @@ type ClickPipeKinesisSource struct {
 }
 
 type ClickPipePostgresSource struct {
-	Host        string                          `json:"host"`
-	Port        int                             `json:"port"`
-	Database    string                          `json:"database"`
-	Credentials *ClickPipeSourceCredentials     `json:"credentials,omitempty"`
-	Settings    ClickPipePostgresSettings       `json:"settings"`
-	Mappings    []ClickPipePostgresTableMapping `json:"tableMappings"`
+	Host                  string                          `json:"host"`
+	Port                  int                             `json:"port"`
+	Database              string                          `json:"database"`
+	Credentials           *ClickPipeSourceCredentials     `json:"credentials,omitempty"`
+	Settings              ClickPipePostgresSettings       `json:"settings"`
+	Mappings              []ClickPipePostgresTableMapping `json:"tableMappings"`
```

Comment:
Should Mappings be `omitempty` too (if we're using same contracts for create and update calls)?